src/test/docker/docker-run-checks.sh

```diff
@@ -238,10 +238,6 @@ else
         -e INCEPTION \
         -e ASAN_OPTIONS \
         -e BUILD_DIR \
-        -e S3_ACCESS_KEY_ID \
-        -e S3_SECRET_ACCESS_KEY \
-        -e S3_HOSTNAME \
-        -e S3_BUCKET \
```

Comment:
The same commit that added these environment variables also added `--network=host` to the docker command  line. That should probably be removed as well (and I vaguely remember it was causing issues recently.)

Reply:
your memory is incredible :-)

I left the `--network=host` in the podman call in `docker-run-systest.sh` for the time being.  It might not be needed, but I wasn't super sure.